src/instructlab/log.py

```diff
@@ -67,7 +71,15 @@ def configure_logging(
         handler.close()
 
     stream = logging.StreamHandler()
-    stream.setFormatter(CustomFormatter(fmt=fmt))
+    try:
+        stream.setFormatter(CustomFormatter(fmt=fmt))
+    except ValueError as e:
+        print(
+            f"Failed to configure log format: {e}\n"
+            "Did you specified a valid log format?\n"
+            "Consider reading: https://docs.python.org/3/library/logging.html#logrecord-attributes"
+        )
+        sys.exit(1)
```

Comment:
Please don't explicitly exit from inside a library function. (It makes it hard to e.g. test it, or reuse in a context where exit is not what we want.) Instead, if you'd like to make `init` fail, propagate the error from `configure_logging` (either by returning a boolean flag or raising an exception), then handle it by the caller.

(That said, I am not sure we should have the `init` option for `log-format` at all, so perhaps the point is moot.)

Reply:
Resolved.